DuckDuckGoTests/WebViewTestHelper.swift

```diff
@@ -123,11 +123,14 @@ class WebKitTestHelper {
     static func preparePrivacyConfig(locallyUnprotected: [String],
                                      tempUnprotected: [String],
                                      contentBlockingEnabled: Bool,
-                                     exceptions: [String]) -> PrivacyConfiguration {
+                                     exceptions: [String],
+                                     httpsEnabled: Bool = false) -> PrivacyConfiguration {
```

Comment:
`httpsEnabled` feels unclear and can be misunderstood, maybe `httpsUpgradesEnabled`?